lib/extension-core.js

```diff
@@ -150,4 +150,26 @@ function core(loader) {
     }
     return loaderInstantiate.call(loader, load);
   }
+
+  function applyExtensions(extensions, loader) {
+    loader._extensions = [];
+    var ext;
+    for(var i = 0, len = extensions.length; i < len; i++) {
+      ext = extensions[i];
+      ext(loader);
+      loader._extensions.push(ext);
+    }
+  }
+
+  loader.clone = function() {
+    var originalLoader = this;
+    var loader = new LoaderPolyfill(originalSystem);
+    loader.baseURL = originalLoader.baseURL;
+    loader.paths = { '*': '*.js' };
+    loader.originalLoader = originalLoader;
```

Comment:
We don't need to, but since we don't have inheritance this would be the way to know which loader is a loader's parent. It might be useful in determining if a loader is a child of the global System, for example.